content/1.7/back-office/international/locations/states/add-edit-state.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Add edit state
+weight: 1
+---
+
+# Add edit state
+
+While adding or editing a state, the following fields / drop-down / toggle are displayed:
+
+ - **Name** - this field is mandatory. A tooltip for this field is displayed: "Provide the state name to be displayed in addresses and on invoices.". When trying to save and if the field is empty, the following error message is displayed: "The name field is required.".
+ - **ISO code** - this field is mandatory. A tooltip for this field is displayed: "1 to 4 letter ISO code. You can prefix it with the country ISO code if needed.". When trying to save and if the field is empty, the following error message is displayed:  "The iso_code field is required."
+ - **Country** - this drop-down is mandatory. A tooltip is displayed: "Country where the state is located. Only the countries with the option "contains state" enabled are displayed." By default "United States" is selected.
+ - **Zone** - this drop-down is mandatory. A tooltip is displayed: "Geographical region where this state is located. Used for shipping". By default the zone is set to: "North America".
```

Comment:
In my case by default Africa is selected.

Reply:
@marwachelly the default country is based on the Prestashop installation. By default, when installing Prestashop from the beginning the standard country is United states. So I am not sure at the moment which country is the correct one and should be mentioned.

Reply:
Well, if the default value depends on another parameter (here it seems to depend on the country chosen during the installation), my recommendation is to not indicate the default value.